source/environments/Pict-Environment-Object.js

```diff
@@ -147,6 +154,14 @@ class PictEnvironmentObject
 		}
 	}
 
+	initializeClassMapLocation(pAddress, pClass)
+	{
+		if (!this.contentMap._CLASS_LIST.indexOf(pClass) > -1)
```

Comment:
```suggestion
		if (this.contentMap._CLASS_LIST.indexOf(pClass) === -1)
```

Or `< 0`. The `!` makes the logic harder to follow.